react/facilitator_controls.js

```diff
@@ -89,33 +89,32 @@ class FacilitatorControls extends Component {
     });
   }
 
-  setActiveTeam1(index, envelopeId) {
+  setActiveTeam1(index, envelopeSlice, j, batchSize) {
```

Comment:
this function, along with `setActiveTeam2`, actually seems more difficult to understand now, since `j` is a pretty ambiguous variable name.  could you update this variable to help others better understand what it represents?

Reply:
also, these two functions are almost identical.  try and see if you can find a way to combine them into a single functions so we don't have so much code repetition here.